types/v56/types.go

```diff
@@ -394,8 +394,8 @@ type AdminVdc struct {
 	Xmlns string `xml:"xmlns,attr"`
 	Vdc
 
-	ResourceGuaranteedMemory float64    `xml:"ResourceGuaranteedMemory,omitempty"`
-	ResourceGuaranteedCpu    float64    `xml:"ResourceGuaranteedCpu,omitempty"`
+	ResourceGuaranteedMemory string     `xml:"ResourceGuaranteedMemory,omitempty"`
```

Comment:
Any reason why these become strings?

Reply:
Yes, I need to serialize `0` and not ignore them.

Reply:
Won't this cause issue like we had with https://github.com/vmware/go-vcloud-director/pull/198 ?

Reply:
In this case no, it just number for how much percentage is guarantee. And don't care - I didn't find solution how to serialize int64=0 -> it's ignored by default, as string="" and similar values

Reply:
What if you skip the `omitempty`?

Reply:
I can't send no value, but should

Reply:
With Dainius we find a little better approach

Reply:
It's pretty cool!